requirements/default.txt

```diff
@@ -207,9 +207,9 @@ django-npm==1.0.0 \
 markus==1.1 \
     --hash=sha256:813ac8cd0eb68bcf7ee618ac2544ce20de5eb0ba27b12f50973c29742800b058 \
     --hash=sha256:821377379e0d225a870996e4dcd1aa5305e4030a8b9b5d8b38dc843cbc30dfa8
-pinax-eventlog==1.1.2 \
-    --hash=sha256:28467f74a9c7ff1b838a43075f46f875daa8a7b37d7dabf097a15b162ef35411 \
-    --hash=sha256:e9c2947fed5773de7cbc610501729d16262a0a913026fad762e4ba940a071258
+pinax-eventlog==2.0.0 \
+    --hash=sha256:ab77944098e6acb0c935a368333f86bf4732106a76a284cb8bc13d3986002245 \
+    --hash=sha256:3261b495ab25842c22e1c38ba3796271f0b7f44db7050d1484eda0148d68fa5d
```

Comment:
pinax-eventlog 2.0.0 fixes the migrations so we don't get "deprecated in django 2.0" warnings from them.